docs/quickstarts/chrome-extension.mdx

```diff
@@ -15,243 +15,238 @@ description: Add authentication and user management to your Chrome Extension wit
     }
   ]}
 >
-
-- Scaffold a new application using the Plasmo framework
-- Install `@clerk/chrome-extension`
-- Set your environment variables
-- Add `<ClerkProvider>` to your application
-- Create a header with Clerk components for users to sign in and out
-- Configure a consistent CRX ID
-- Load your Chrome Extension into your Chromium-based browser
-- Test your Chrome Extension
-
+  - Scaffold a new application using the Plasmo framework
+  - Install `@clerk/chrome-extension`
+  - Set your environment variables
+  - Add `<ClerkProvider>` to your application
+  - Create a header with Clerk components for users to sign in and out
+  - Configure a consistent CRX ID
+  - Load your Chrome Extension into your Chromium-based browser
+  - Test your Chrome Extension
 </TutorialHero>
 
 <Steps>
+  ## Configure your authentication options
 
-## Configure your authentication options
-
-Chrome Extensions can be used just a popup, a side panel or in conjunction with a web application. Popups and side panels have limited authentication options. See [Chrome Extension Authentication Options](/docs/references/chrome-extension/overview#authenication-options) for information on what options are available. Use the table to ensure you have the correct authentication options enabled/disabled in your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=authentication).
-
-## Scaffold a new application using the Plasmo framework
-
-[Plasmo](https://docs.plasmo.com/framework) is a feature rich Browser Extension framework, which includes hot reloading, creating development and production extension builds easily from the same code, and more.
-
+  Chrome Extensions can be used just a popup, a side panel, or in conjunction with a web application. Popups and side panels have limited authentication options. [Learn more about what options are available.](/docs/references/chrome-extension/overview#authenication-options)
```

Comment:
I'm not sure I like removing the following, as I think this will be a confusing area for customers and I want to really draw attention to it. Happy to talk about options too -- I just want to make sure this is highlighted enough. Do you feel the revised copy is really strong on this point?

---
Use the table to ensure you have the correct authentication options enabled/disabled in your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=authentication).